From f14979632e13e03e5ebb9abc29b7aeb1bcf84f54 Mon Sep 17 00:00:00 2001
From: "Chew, Chiau Ee" <chiau.ee.chew@intel.com>
Date: Wed, 3 Oct 2018 14:34:38 +0800
Subject: [PATCH] efi/Makefile: remove standard system header file directory

Although the -nostdinc option is enabled, but the standard system header
file directory (host) is still not being ignored during compilation.
Thus, remove the -I hardcoded path from Makefile. The full custom path is
being passed in via the fwupdate recipe using EXTRA_OEMAKE.

The associated build errors as below:
ccl: error: include location "/usr/include/efi/" is unsafe for cross-compilation [-Werror=poison-system-directories]
ccl: error: include location "/usr/include/efi/x86-64/" is unsafe for cross-compilation [-Werror=poison-system-directories]

Upstream-Status: Inappropriate [native]

Signed-off-by: Chew, Chiau Ee <chiau.ee.chew@intel.com>
---
 efi/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/efi/Makefile b/efi/Makefile
index 3d7aed9..85b8e1c 100644
--- a/efi/Makefile
+++ b/efi/Makefile
@@ -15,7 +15,6 @@ CFLAGS	?= -Og -g3 -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 \
 BUILDFLAGS	:= $(CFLAGS) -fpic -Werror -Wall -Wextra -fshort-wchar \
 	-fno-merge-constants -ffreestanding \
 	-fno-stack-protector -fno-stack-check --std=gnu11 -DCONFIG_$(ARCH) \
-	-I/usr/include/efi/ -I/usr/include/efi/$(ARCH)/ \
 	-iquote$(TOP)/include "-DDEBUGDIR=L\"$(DEBUGDIR)/\""
 CCLDFLAGS	?= -nostdlib -Wl,--warn-common \
 	-Wl,--no-undefined -Wl,--fatal-warnings \
-- 
2.7.4

